go/vendor/github.com/keybase/go-crypto/rsa/pkcs1v15.go

```diff
@@ -1,325 +0,0 @@
-// Copyright 2009 The Go Authors. All rights reserved.
```

Comment:
We have to leave this in.  I forked `go-crypto/rsa` to get support for large-(ish) *e* exponents in RSA.  See [this issue](https://github.com/keybase/keybase-issues/issues/2000) and the PRs linked to.